data/json/vehicleparts/motor.json

```diff
@@ -85,14 +82,16 @@
     "item": "motor_large",
     "difficulty": 3,
     "durability": 400,
-    "power": 149200,
-    "energy_consumption": 166000,
+    "power": 138000,
```

Comment:
Please make this 150,000 W of power at 165,000 W of consumption.

Reply:
I can update one or both of the numbers here, but I would like to keep the theme of efficiency rising as the motor sizes rise. This is true of real electric motors (at least on average). The regular motor has an efficiency of 90%, and the enhanced has 94%, so I would like to keep the large at or around 92%.

Reply:
To keep that efficiency, I could do 150k/163k, or 151.8k/165k. Do you have a preference?

Reply:
150K, 163K.